### PR TITLE
fix issue in the load_configuration() preventing one to set objects that were not records or arrays

### PR DESCRIPTION
**Pull request by af-silva**
The previous implementation only allow one to set composed objects, this is objects with sub-index's. Now we can set all the supported objects loaded by the EDS file

### CODE REVIEW

File: canopen/node/remote.py [hunk 1]
```diff
@@ -89,27 +89,46 @@ def restore(self, subindex=1):
...
+        :param name: Object name
+        :param value: Value to set in the object
+        """
+        try:
+            if subindex is not None:
+                logger.info(str('SDO [{index}][{subindex}]: {name}: {value}'.format(
+                    index=format(index, '#04x'),
+                    subindex=subindex,
+                    name=name,
+                    value=format(value, '#04x'))))
```
Comment (christiansandberg) [inline canopen/node/remote.py]: You could include the format of the hex in the first string: `logger.info('SDO [{index:#06x}][{subindex}]: {name}: {value:#06x}' ...`.

File: canopen/node/remote.py [hunk 2]
```diff
@@ -89,27 +89,46 @@ def restore(self, subindex=1):
...
-                        except canopen.SdoAbortedError as e:
-                            # WORKAROUND for broken implementations: the SDO is set but the error
-                            # "Attempt to write a read-only object" is raised any way.
-                            if e.code != 0x06010002:
-                                # Abort codes other than "Attempt to write a read-only object"
-                                # should still be reported.
-                                print('[ERROR SETTING object {0}:{1}]  {2}'.format(subobj.index, subobj.subindex, str(e)))
-                                logger.info(str(e))
-                                raise
+                    if isinstance(subobj, Variable) and (subobj.access_type in ('rw', 'rww')) and (subobj.value is not None):
```
Comment (christiansandberg) [inline canopen/node/remote.py]: I've added the `.writable` attribute to Variable as a higher level API than `.access_type`.

File: canopen/node/remote.py [hunk 3]
```diff
@@ -89,27 +89,46 @@ def restore(self, subindex=1):
...
+        except canopen.SdoCommunicationError as e:
+            logger.info(str(e))
+        except canopen.SdoAbortedError as e:
+            # WORKAROUND for broken implementations: the SDO is set but the error
+            # "Attempt to write a read-only object" is raised any way.
+            if e.code != 0x06010002:
+                # Abort codes other than "Attempt to write a read-only object"
+                # should still be reported.
+                logger.warning('[ERROR SETTING object {0}:{1}]  {2}'.format(index, subindex, str(e)))
+                logger.info(str(e))
```
[the remark this answers is not in the document]
Comment (af-silva) [reply canopen/node/remote.py]: You are right, removed!

File: canopen/node/remote.py (hunk 1 above)
Comment (af-silva) [reply canopen/node/remote.py]: Ok, done!

File: canopen/node/remote.py (hunk 2 above)
Comment (af-silva) [reply canopen/node/remote.py]: Ok, it's more clean this way. Done!

**Pull request merged**